### ISSUE TITLE
What do I need to input when running prompt_generator.py

### ISSUE THREAD

**Issue by Zhangky2025** [AmirAbaskohi/LM-CPPF#1]
Hello!
I noticed there's a line of code in the 'prompt_generator.py' file that requires user input. 
`
paraphrased = input(f"Paraphrase this: {sample_row['text']}")
`
What do I need to input when running this file?
I would appreciate your help in this issue.

**Comment by AmirAbaskohi** [comment]
Hello,

This script is designed to generate paraphrase prompts within a few-shot setting. Previously, we relied on paraphrased data from various datasets generated using QuillBot. However, since we haven't shared this specific data, I've modified the script to prompt users to input their paraphrased samples.

Feel free to enter the paraphrased version of the sample displayed on your screen. Keep in mind that if you plan to use this on a larger scale, manually inputting each paraphrase may become cumbersome. I recommend customizing this part of the code to suit your needs, whether by connecting to QuillBot, as we have done, or by integrating another model.

I'll include a comment in the code to inform others about this modification.

Thank you for bringing this to my attention.
